SYCL/Sampler/normalized-clampedge-linear-float.cpp

```diff
@@ -2,12 +2,12 @@
 // RUN: %HOST_RUN_PLACEHOLDER %t.out %HOST_CHECK_PLACEHOLDER
 // RUN: %GPU_RUN_PLACEHOLDER %t.out %GPU_CHECK_PLACEHOLDER
 // RUN: %CPU_RUN_PLACEHOLDER %t.out %CPU_CHECK_PLACEHOLDER
-// XFAIL: cpu
 // XFAIL: cuda
 // UNSUPPORTED: level_zero && windows
 
-// CPU failing all linear interpolation at moment. Waiting on fix.
-// CUDA failing all linear interpolation at moment. Waiting on fix.
+// CUDA works with image_channel_type::fp32, but not with any 8-bit per channel
```

Comment:
Is it missing support or a bug? may be it is reasonable to move image_channel_type::fp32 to separate file to be able test samplers on CUDA?

Reply:
@vladimirlaz  - I could do that, but there are some 8 different tests of linear filtering, so that would be 8 additional files to test CUDA.    These tests aren't going to stay here forever. Ultimately, when the SYCL-CTS are updated to have Sampler coverage, we'll be removing these lit tests. These are here right now primarily to avoid regressions from the  different teams working on sampler reports ( OpenCL, L0, SYCL RT, etc)

If you want me to break out these CUDA cases to 8 new files, let me know and I'll do so.